docs/source/tensorbay_cli/getting_started_with_cli.rst

```diff
@@ -105,3 +105,41 @@ CLI: Commit the Draft
 .. code:: html
 
    $ gas commit tb:<dataset_name>#<draft_number> [-m <message>]
+
+***************
+Profile
+***************
+
+For users with multi TensorBay accounts or different workspaces,
+CLI provides profiles to easily authenticate and use different accessKeys.
+
+Set the accessKey into the specific profile:
+
+.. code:: html
+
+   $ gas -p <profile_name> auth [ACCESSKEY]
+
+Use the specific profile's accessKey to execute a command:
+
+.. code:: html
+
+   $ gas -p <profile_name> <command>
+
+For example, show the specific profile's authentication information:
+
+.. code:: html
+
+   $ gas -p <profile_name> auth --get
+
+Use the specific accessKey which not in the profiles to execute a command:
+
+.. code:: html
+
+   $ gas -k <Accesskey> <command>
+
+For example, list the segments of a dataset with specific accessKey.(default branch):
```

Comment:
No need to use `default branch` here.
`with specific accessKey` -> `with the given accessKey`?